packages/command/src/docOpts.ts

```diff
@@ -142,13 +140,14 @@ export class DocOpts<T extends typeof Command> {
     return this.flagList;
   }
 
-  private generateUsageElement(elementStrs: string[] = [], flagGroups: flagType[], beforeChar = '', afterChar = '') {
+  private generateUsageElement(elementStrs: string[] = [], flagGroups: FlagsType, beforeChar = '', afterChar = '') {
     for (const flag of flagGroups) {
       // don't show usage for hidden flags
       if (!flag.hidden) {
+        const kind = ensure(getString(flag, 'kind'));
```

Comment:
I think this should always be defined for `SfdxCommand` flags, including the ones defined by oclif, so the `ensure` should be safe.  Worth making sure of that though.

Reply:
We could safely get at more sfdx-specific variables here (like min, max, and so on) by using the `get` or `has` fns.